src/main/java/redis/clients/jedis/commands/BinaryJedisCommands.java

```diff
@@ -482,5 +493,9 @@ default String setex(byte[] key, int seconds, byte[] value) {
   @Deprecated
   List<StreamConsumersInfo> xinfoConsumers(byte[] key, byte[] group);
 
-  List<Object> xinfoConsumersBinary(byte[] key, byte[] group);
+  List<Object> xinfoConsumersBinary(byte[] key, byte[] group);;
```

Comment:
Extra semicolon at the end of the line
```suggestion
  List<Object> xinfoConsumersBinary(byte[] key, byte[] group);
```